lib/msf/core/web_services/authentication/strategies/api_token.rb

```diff
@@ -48,6 +53,14 @@ def validate_user(user)
         !user.nil? ? {valid: true, code: 0, message: nil} : {valid: false, code: 401, message: "Invalid API token."}
       end
 
+      # Authenticates the API token from a configuration file
+      def auth_from_env(token)
+        if token == request.env['msf.api_token']
+          success!(message: "Successful auth from file token")
```

Comment:
The message wasn't updated when authentication shifted from a file API token to the environment API token.